components/tikv_util/src/sys/cpu_time.rs

```diff
@@ -126,16 +86,17 @@ mod imp {
             let mut num_cpus_u = 0;
             let mut cpu_info = ptr::null_mut();
             let mut msg_type = 0;
-            let err = host_processor_info(
+            let ret = host_processor_info(
```

Comment:
I think you can get these APIs from libc without any build scripts. https://docs.rs/libc/latest/x86_64-apple-darwin/libc/index.html

Reply:
libc doesn't contain all my needs, e.g. `task_threads`.

Reply:
Any other symbols it misses? Just defining `task_threads` seems a lot simpler and lightweight than using a build script.